frontend-react/src/pages/daily/Table/ReportsTable.tsx

```diff
@@ -116,11 +116,14 @@ function ReportsTable({ sortBy }: { sortBy?: string }) {
             <div className="grid-col-12">
                 <Table config={resultsTableConfig} filterManager={fm} />
             </div>
-            <div className="grid-col-12">
-                {reports.filter((report) => report.receivingOrgSvc === chosen)
-                    .length === 0 ? (
-                    <p>No results</p>
-                ) : null}
+            <div className="grid-container margin-bottom-10">
```

Comment:
Just as a note, in my free time I've been trying to architect a pattern that makes sense so we don't have rogue grid-containers floating around solving for this padding issue. I'll spin up a ticket to track that so we can have a much more predictable container.